subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractToolingApiCrossVersionPerformanceTest.groovy

```diff
@@ -300,32 +274,19 @@ abstract class AbstractToolingApiCrossVersionPerformanceTest extends Specificati
                             omit = true
                         }
                     }
-                    if (experimentSpec.listener) {
-                        experimentSpec.listener.afterInvocation(info, measuredOperation, cb)
-                    }
                     if (!omit) {
                         versionResults.add(measuredOperation)
                     }
                 }
             }
         }
 
-        private void warmup(toolingApi, File workingDir) {
-            OperationTimer timer = new OperationTimer()
-            experimentSpec.with {
+        private void warmup(toolingApi, ToolingApiBuildExperimentSpec experimentSpec) {
+            experiment.with {
                 def count = iterationCount("warmups", warmUpCount)
                 count.times { n ->
-                    BuildExperimentInvocationInfo info = new DefaultBuildExperimentInvocationInfo(experimentSpec, workingDir, BuildExperimentRunner.Phase.WARMUP, n + 1, count)
-                    if (experimentSpec.listener) {
-                        experimentSpec.listener.beforeInvocation(info)
```

Comment:
Does that mean we don't support the listener for tooling api performance tests?

Reply:
It wasn't used, so I simplified. It's easy enough to add when needed.